### PR TITLE
Accessibility testing

### PR DESCRIPTION
**Pull request by contolini**
Adds automated pa11y test to be run by Travis CI.
## Additions
- Adds `npm test` script.
- Adds config file for Travis CI.
- Adds Travis CI badge to readme.
## Testing
- Pull down changes.
- `npm install`
- `npm test`
## Review
- @anselmbradford

### CODE REVIEW

File: custom-demo/static/css/custom-demo.css [hunk 1]
```diff
@@ -64,7 +64,7 @@ audio:not([controls]) {
 }
 /**
  * Address `[hidden]` styling not present in IE 8/9/10.
- * Hide the `template` element in IE 8/9/11, Safari, and Firefox < 22.
+ * Hide the `template` element in IE 8/9/10/11, Safari, and Firefox < 22.
```
Comment (anselmbradford) [inline custom-demo/static/css/custom-demo.css:67]: Any Safari version info available?

**Comment by anselmbradford** [comment]
:+1: Passes the tests.

File: custom-demo/static/css/custom-demo.css (hunk 1 above)
Comment (contolini) [inline custom-demo/static/css/custom-demo.css:67]: That text is from [normalize.css](https://github.com/necolas/normalize.css/blob/master/normalize.css#L74) so there's not much we can do to change it.

Comment (anselmbradford) [inline custom-demo/static/css/custom-demo.css:67]: :+1:

**Pull request merged**